samples/python/resize_pil.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+'''
+Compare resize function with interpolations: cv.INTER_NEAREST, cv.INTER_NEAREST_PIL, PIL.Image.NEAREST
+
+Usage:
+  resize_pil.py [<image>]
+
+'''
+
+# Python 2/3 compatibility
+from __future__ import print_function
+
+import numpy as np
+import cv2 as cv
+
+import sys
+
+from PIL import Image
+
+def main():
+
+    try:
+        fn = sys.argv[1]
+    except:
+        fn = 'lena.jpg'
+
+    img_cv_lena = cv.imread(cv.samples.findFile(fn), cv.IMREAD_GRAYSCALE)
+
+    if img_cv_lena is None:
+        print('Failed to load image file:', fn)
+        sys.exit(1)
+
+    img_pil_lena = Image.fromarray(np.array(img_cv_lena))
+
+    img_cv_lena_resized = cv.resize(img_cv_lena, (231,322), interpolation=cv.INTER_NEAREST)
+    img_pil_lena_resized = img_pil_lena.resize((231,322), Image.NEAREST)
+    assert (np.array(img_cv_lena_resized) != np.array(img_pil_lena_resized)).any()
```

Comment:
Why do we need assert?

Reply:
I wanted to note that cv.INTER_NEAREST differs from  Image.NEAREST

Reply:
But it will stop script execution here with `AssertionError` and traceback

Reply:
I tested it before and I've tested it once again. 
assert (np.array(img_cv_lena_resized) **!=** np.array(img_pil_lena_resized)).**any()**
As cv.INTER_NEAREST really differs from Image.NEAREST we really have difference at least in in one pixel. So it isn't stop, goes further and print 
_cv.INTER_NEAREST is not equal to Image.NEAREST_